ddtrace/profile/exporter/http.py

```diff
@@ -0,0 +1,187 @@
+import binascii
+import datetime
+import gzip
+import logging
+import os
+import platform
+import uuid
+
+from ddtrace.vendor import six
+from ddtrace.vendor.six.moves import http_client
+from ddtrace.vendor.six.moves.urllib import parse as urlparse
+
+import ddtrace
+from ddtrace.profile import _traceback
+from ddtrace.profile import exporter
+from ddtrace.vendor import attr
+from ddtrace.profile.exporter import pprof
+
+
+LOG = logging.getLogger(__name__)
+
+
+RUNTIME_ID = str(uuid.uuid4()).encode()
+HOSTNAME = platform.node()
```

Comment:
how does this differ from `socket.gethostname()`? 

https://github.com/DataDog/dd-trace-py/blob/7125e740b0303d32dc4e2719c063f3505376094c/ddtrace/internal/hostname.py#L20